ui/src/extractedTranslations/zh-hk/translation.json

```diff
@@ -0,0 +1,308 @@
+          "next_action": "下一步",
+          "title": "選擇想移動的相片"
+        },
+        "title": "移動相片臉孔"
+      }
+    },
+    "recognize_unlabeled_faces_button": "已識別的未命名臉孔",
+    "tableselect_face_group": {
+      "search_faces_placeholder": "搜尋臉孔..."
+    },
+    "tableselect_image_faces": {
+      "search_images_placeholder": "搜尋相片..."
+    }
+  },
+  "photos_page": {
+    "title": "相片"
+  },
+  "places_page": {
+    "title": "地點"
+  },
+  "routes": {
+    "page_not_found": "找不到網頁"
+  },
+  "settings": {
+    "concurrent_workers": {
+      "description": "最多同時進行的掃描工作數目",
+      "title": "同時掃描工作"
+    },
+    "logout": "登出",
+    "periodic_scanner": {
+      "checkbox_label": "啟用週期性掃描",
+      "field": {
+        "description": "對全部使用者自動掃描的頻率",
+        "label": "掃描頻率"
+      },
+      "interval_unit": {
+        "days": "日",
+        "hour": "小時",
+        "minutes": "分鐘",
+        "months": "月",
+        "seconds": "秒"
+      },
+      "title": "週期性掃描"
+    },
+    "scanner": {
+      "description": "將掃描所有使用者的新媒體或更新媒體",
+      "scan_all_users": "掃描所有使用者",
+      "title": "掃描"
+    },
+    "user_preferences": {
+      "change_language": {
+        "description": "Change website language specific for this user",
+        "label": "Website language"
+      },
+      "language_selector": {
+        "placeholder": "Select language"
+      },
+      "title": "使用者選項"
+    },
+    "users": {
+      "add_user": {
+        "submit": "新增使用者"
+      },
+      "confirm_delete_user": {
+        "action": "刪除 {{user}}",
+        "description": "<0>確定刪除 <1></1>?</0><p>此操作無法取消</p>",
+        "title": "刪除使用者"
+      },
+      "password_reset": {
+        "description": "更改密碼 for <1></1>",
+        "form": {
+          "label": "新密碼",
+          "placeholder": "密碼",
+          "submit": "提交"
+        },
+        "title": "更改密碼"
+      },
+      "table": {
+        "column_names": {
+          "action": "動作",
+          "capabilities": "能力",
+          "photo_path": "相簿路徑",
+          "username": "使用者名稱"
+        },
+        "new_user": "新增使用者",
+        "row": {
+          "action": {
+            "change_password": "更改密碼",
+            "delete": "刪除",
+            "edit": "更改",
+            "scan": "掃描"
+          }
+        }
+      },
+      "title": "使用者"
+    },
+    "version_info": {
+      "build_date_title": "編譯日期",
+      "title": "Photoview 版本",
+      "version_title": "釋出版本"
+    }
+  },
+  "share_page": {
+    "media": {
+      "title": "已分享媒體"
+    },
+    "protected_share": {
+      "description": "此分享受密碼保護",
+      "password_required_error": "需要密碼",
+      "title": "已保護的分享"
+    },
+    "share_not_found": "找不到分享",
+    "share_not_found_description": "可能此分享已過期或已刪除",
+    "wrong_password": "密碼錯誤，請重試"
+  },
+  "sidebar": {
+    "album": {
+      "title_placeholder": "相簿名稱"
+    },
+    "download": {
+      "filesize": {
+        "byte": "",
+        "giga_byte": "",
+        "kilo_byte": "",
+        "mega_byte": "",
+        "tera_byte": "",
```

Comment:
Please make sure to write null instead of an empty string, this way it will fall back to the original (english) translation, with an empty string nothing is shown.

Reply:
Fixed!